Makefile

```diff
@@ -59,6 +60,9 @@ COMPONENTS_USAGE_STATS_FILES = $(shell \
 		-type f \
 		\( -name '*.js' -or -name '*.jsx' \) \
 )
+COMPONENTS_PROPTYPE_FILES = $(shell \
+	find client -name 'index.jsx' -or -name 'example.jsx' \
```

Comment:
There's some components in the devdocs that are outside of the components and blocks folders:

- my-sites/post-selector/docs/example
- lib/sites-list/docs/example
- my-sites/plan-storage/docs/example

for example ...